src/modules/tools/spindle/SoftSerial/SoftSerial.h

```diff
@@ -13,7 +13,7 @@
 /** A software serial implementation
  *
  */
-class SoftSerial: public Stream {

```

Comment:
won't this break other things using SoftSerial?? I think you need to leave it in.

Reply:
Never mind ;)